src/octopeer-github/main/Logger.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by Maarten on 17-05-2016.
+ */
+
+const Logger = new (class Logger {
+
+    private isDebug = false;
+
+    public log(obj: any) {
```

Comment:
I would prefer if this would be refactored to `warn()`. Then you can refactor `_log()` to `log()`. That would give more clarity in your code.